development/site-components/sm-buttons.pug

```diff
@@ -0,0 +1,15 @@
+html
+	div.media-bar(id="followMe" name="about")
+		a(name="sm-buttons")
+		//- <!--Instagram -->
```

Comment:
Again, this applies to all comments in jade. Lose the `<!--some comment -->`. This comment will not show up in your html so you can simply write: 
`//- INSTAGRAM` 
or if you want it to show up as a comment in html (viewable only in the source code)
`// INSTATGRAM` jade will convert if for you automatically without the `<!-- ... -->`